airbyte-cdk/python/airbyte_cdk/sources/declarative/checks/check_stream.py

```diff
@@ -39,7 +39,7 @@ def check_connection(self, source: Source, logger: logging.Logger, config: Mappi
                     records = stream.read_records(sync_mode=SyncMode.full_refresh)
                     next(records)
                 except Exception as error:
-                    return False, f"Unable to connect to stream {stream} - {error}"
+                    return False, f"Unable to connect to stream {stream_name} - {error}"
```

Comment:
for declarative yaml files we end up printing the entire yaml config in the UI which is pretty ugly and makes it hard to see the real error message